clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/AbstractStorageEngine.java

```diff
@@ -579,7 +579,8 @@ public synchronized Optional<OffsetRecord> getPartitionOffset(int partitionId) {
    */
   public synchronized void clearPartitionOffset(int partitionId) {
     if (!metadataPartitionCreated()) {
-      throw new StorageInitializationException("Metadata partition not created!");
+      LOGGER.info("Metadata partition not created; there is nothing to clear!");
```

Comment:
can you add more info in the log like resource name/partition etc.

Reply:
Good call! Updated.